mmv1/third_party/terraform/services/iamworkforcepool/resource_iam_workforce_pool_provider_test.go.erb

```diff
@@ -372,3 +372,109 @@ resource "google_iam_workforce_pool" "my_pool" {
 }
 `, context)
 }
+
+<% unless version == 'ga' -%>
+func TestAccIAMWorkforcePoolWorkforcePoolProvider_oidc_beta(t *testing.T) {
```

Comment:
How long do these resources take to create? If it's longer than ~1min, it would be great to merge this into an existing update test (if one exists)

Reply:
I think it will take about 1 to 2 mins? I am not sure if it is a good idea to merge the new tests into existing update tests since they test different flows.

Reply:
This test can probably be deleted - it looks like it duplicates the test generated from the example file.

Reply:
Done.